Makefile

```diff
@@ -185,6 +186,16 @@ docker-build-build: $(shell docker-required --path . --replace-pattern='^test(.*
 		$(APPS_FILES_ALT)
 	docker build --build-arg=VERSION=$(VERSION) --tag=$(DOCKER_BASE)-build:$(MAJOR_VERSION) .
 
+docker/qgisserver/commons: commons
+	rm --recursive --force $@
+	cp --recursive $< $@
+	touch $@
```

Comment:
I find simpler to use repository root as docker context

Reply:
No, we relay don't wants to mix everything ...